# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../p3nfs/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- p3nfs-5.18/nfsd/mp_mount.c.vanilla	2005-05-20 23:56:43.000000000 +0200
+++ p3nfs-5.18/nfsd/mp_mount.c	2005-05-20 23:56:50.000000000 +0200
@@ -60,6 +60,7 @@
 #include <rpc/rpc.h>
 #ifdef linux
 #define _LINUX_SUNRPC_MSGPROT_H_
+#define _LINUX_STRING_H_
 #define __KERNEL__		/* define nfs_fh for nfs_mount_data */
 #include "nfs_prot.h"
 #undef __KERNEL__
